src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -108,7 +108,6 @@ private boolean runPolling() {
 
             public void run() {
                 if (runPolling()) {
-                    String name = " #" + job.getNextBuildNumber();
```

Comment:
I was concerned that for flyweight tasks merely calling `scheduleBuild` would cause the build to start _immediately_ and thus produce an inaccurate estimate if this variable were inlined, but the following test confirms that starting is asynchronous:

```java
@Test
public void nextBuildNumber() throws Exception {
    WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
    p.setDefinition(new CpsFlowDefinition(""));
    assertEquals(1, p.getNextBuildNumber());
    QueueTaskFuture<WorkflowRun> b = p.scheduleBuild2(0);
    assertEquals(1, p.getNextBuildNumber());
    b.get();
    assertEquals(2, p.getNextBuildNumber());
}
```